README.md

```diff
@@ -0,0 +1,25 @@
+# Nookpendium
+
+-----------------------
+
+Nookpendium is an Android application developed in Kotlin with the main goal of providing Animal Crossing players a one-stop shop for all in-game resource knowledge and tracking.
+
+In it's current state, the app only supports tracking critters for Animal Crossing: New Horizons (Switch) and Animal Crossing: New Leaf, but it's planned to eventually support all Animal Crossing games with varying levels of information and tracking capabilities tailored to each version of the game.
+
+## Screenshots
+
+--------------
+
+![](PromoImage (1).jpeg)  ![](PromoImage (2).jpeg) ![](PromoImage (3).jpeg)
+
+
+
+## How to install
+
+-----
+
+The app can currently be installed in three different ways:
+
+- Either by downloading it from the Google Play Store
+- By downloading the latest stable version APK from the release tab here on GitHub
```

Comment:
You can upload a release on github and link to /releases/latest here instead of explaining this